doc/firstWeek summary.md

```diff
@@ -0,0 +1,25 @@
+## First week summary with question and answers.
+
+### Question- Why OAM users needs GItOps? 
+
+> OAM users needs GitOps because of multiple reasons some of them listed below -
+- <b>Deploy Faster More Often</b> When we say “high velocity” we mean that every product team can safely ship updates many times a day — deploy instantly, observe the results in real time, and use this feedback to roll forward or back.
+- <b>Easy and Fast Error Recovery</b> — The Git record is then not just an audit log but also a transaction log. You can roll back & forth to any snapshot.
```

Comment:
Error Recovery is not something that GitOps does. It is something that user who can do with or without GitOps. You need to demonstrate how GitOps help with Error Recovery. For example, GitOps provides auditable and versionable commit history, which provides a safe and familiar process to help user to rollout and rollback